DataCollection/src/Domain/Managers/WaterbirdForagingManager.cs

```diff
@@ -203,15 +204,12 @@ private void LoadModifiedEntitiesIntoPersistenceLayer(SurveyPending survey)
                 //Set the state to Modified only if the object is already created.
                 if (x.Id > 0)
                 {
-                    SetModifiedState(_persistence, x);
+                    _persistence.SetModifiedStateOn(x);
                 }
                 _persistence.SaveChanges();
             });
         }
 
-        public static Action<ISurveyPersistence, object> SetModifiedState = (ISurveyPersistence persistenceLayer, object input) =>
-        {
-            persistenceLayer.Entry(input).State = System.Data.Entity.EntityState.Modified;
-        };
```

Comment:
Moved this to an extension method so that all persistence classes can use this as shared code.